core/templates/pages/exploration-editor-page/exploration-editor-page.component.ts

```diff
@@ -275,6 +280,7 @@ angular.module('oppia').component('explorationEditorPage', {
           ExplorationCorrectnessFeedbackService.init(
             explorationData.correctness_feedback_enabled);
 
+          ctrl.alertsService = AlertsService;
```

Comment:
why this change?

Reply:
Removed it as it was needed before but not now.